changelog/1.21.0.md

```diff
@@ -5,7 +5,8 @@ description: "Released on 21/07/2021"
 
 ### Breaking changes ❗
 
-There are no breaking changes in 1.21.0.
+- web: The workspace provider ping functionality has been removed from the Coder
+  UI.
```

Comment:
This is not a breaking change, legacy workspace providers still have the ping indicator. The ping indicator is only missing for net v2 enabled providers.